src/main/java/com/amazon/opendistroforelasticsearch/security/OpenDistroSecurityPlugin.java

```diff
@@ -491,10 +494,10 @@ public void onIndexModule(IndexModule indexModule) {
         //called for every index!
 
         if (!disabled && !client && !openDistroSSLConfig.isSslOnlyMode()) {
-            log.debug("Handle dlsFlsAvailable: "+dlsFlsAvailable+"/auditLog="+auditLog.getClass()+" for onIndexModule() of index "+indexModule.getIndex().getName());
-            if (dlsFlsAvailable) {
+            log.debug("Handle auditLog {} for onIndexModule() of index {}", auditLog.getClass(), indexModule.getIndex().getName());
+            if (advancedModulesEnabled) {
```

Comment:
Can this changed to ! isSslOnlyMode() instead relying on advancedModuleEnabled?

Reply:
Removed as it is already shortcircuited by `!openDistroSSLConfig.isSslOnlyMode()`